dev-libs/qtcompress/qtcompress-0_p20180513.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils toolchain-funcs virtualx
+
+DESCRIPTION="A public API for QZip in an easy to use module"
+
+HOMEPAGE="https://github.com/nezticle/qtcompress"
+MY_COMMIT="23f8831826cd72aedf99fc3699148b6c994fd677"
+SRC_URI="https://github.com/nezticle/qtcompress/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+LICENSE="LGPL-2.1"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+S="${WORKDIR}/qtcompress-${MY_COMMIT}"
+
+RDEPEND="
+	dev-qt/qtcore:5
+	sys-libs/zlib:=
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	virtual/pkgconfig
+	test? ( dev-qt/qttest:5 )
```

Comment:
This is a rather interesting package because it installs tons of headers and few libraries, but no binaries.

Reply:
Okay?

Reply:
I see what you mean. I've put it to BDEPEND because python eclasses put `test? ( ${RDEPEND} )` into BDEPEND instead of DEPEND. Because cross-compilation isn't compatible with tests without something like qemu-user

Reply:
It's got something to do with how python executables import their modules at runtime which is not the case with compiled products. AFAIK Perl has similar issues.

https://qa-reports.gentoo.org/output/genrdeps/dindex/dev-qt/qttest
https://qa-reports.gentoo.org/output/genrdeps/bindex/dev-qt/qttest

Reply:
I see. Done.